### ISSUE TITLE
Predefined dashboard as "community template" for major open source tools

### ISSUE THREAD

**Issue by prigal** [metabase/metabase#11663]
_NB : I hope it's not a duplicate, I have searched for this kind of feature request, but I did not find anything._

**Is your feature request related to a problem? Please describe.**
Most of metabase users are open source fan. They often use metabase to vizualise data from other open source tools. I'm in the ecommerce industry, there may be hundreds of other people like me who are building dashboard for magento, prestashop, sylius or other cart tools. 

Why not putting our efforts in common to build standard pre-packaged dashboard for major tools. In addition of helping community, that will help metabase adoption I think.

**Describe the solution you'd like**
Google do this for analytics, you can add to your Google analytics some dashboard built by the community. This can be an idea but that mean managing a central server for providing templates. Other idea can be adding this through a plugin system, plugins could be directly grabbed from github. 

**Describe alternatives you've considered**
Custom sql shared via a forum ? Not really a solution.

**How important is this feature to you?**
Not that important for me as long as my dashboard are already built. But I really think it can help metabase adoption. Most open source tools add some kind of reports in it, but it's not their main purpose and work is often not really good. Metabase can help other tools looking better and achieve business intelligence correctly. 

**Additional context**
n/a

**Comment by flamber** [comment]
This would probably depend on #2052 (perhaps even duplicate of that).
